Composer/packages/server/src/utility/electronContext.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import { ElectronAuthParameters } from '@botframework-composer/types';
+
+export type ElectronContext = {
+  getAccessToken: (
+    params: ElectronAuthParameters
+  ) => Promise<{ accessToken: string; acquiredAt: number; expiryTime: number }>;
+};
+
+let context;
+
+export const useElectronContext = (): ElectronContext => context;
```

Comment:
Can we just use property accessors for setting and using?

let obj = { _get propName()_ { // getter, the code executed on getting obj.propName }, _set propName(value)_ { // setter, the code executed on setting obj.propName = value } };?